pg/array.go

```diff
@@ -0,0 +1,118 @@
+package pg
+
+import (
+	"bytes"
+	"fmt"
+	"reflect"
+	"strconv"
+
+	"github.com/elgris/sqrl"
+)
+
+// Array converts value into Postgres Array
+func Array(arr interface{}) sqrl.Sqlizer {
+	return array{arr}
+}
+
+// ToSql builds the query into a SQL string and bound args.
+func (a array) ToSql() (string, []interface{}, error) {
+	if err := checkArrayType(a.value); err != nil {
+		return "", nil, err
+	}
+
+	buf := &bytes.Buffer{}
+	marshalArray(a.value, buf)
+	return "?", []interface{}{buf.String()}, nil
+}
+
+var validElems = map[reflect.Kind]struct{}{reflect.Int: {}, reflect.Float32: {}, reflect.Float64: {}, reflect.String: {}}
```

Comment:
2 comments here:
1. Better to split this big line into multiline definition to improve readability. Like this:
```
var validElems = map[reflect.Kind]struct{}{
	reflect.Int:     {},
	reflect.Float32: {},
	reflect.Float64: {},
	reflect.String:  {},
}      
```
2. What about integer types defined in [`reflect` package](https://golang.org/src/reflect/type.go?s=8213:8227#L216)?